src/Symfony/Component/Console/Application.php

```diff
@@ -138,6 +138,9 @@ public function run(InputInterface $input = null, OutputInterface $output = null
                 $e = null;
                 $exitCode = 0;
```

Comment:
Yes, that's the conclusion we came to with @chalasr regarding this part. I guess now it's better to wait for a PR with a bigger vision of the remaining issues with the component, as @nicolas-grekas seems to plan.